fastchat/llm_judge/show_result.py

```diff
@@ -37,6 +38,17 @@ def display_result_single(args):
         df_3 = df[["model", "score"]].groupby(["model"]).mean()
         print(df_3.sort_values(by="score", ascending=False))
 
+        print("\n########## Category Average ##########")
+        df_category = df_all[["model", "score", "question_id", "turn"]]
+        df_category = df_category[df_category["score"] != -1]
+        if args.model_list is not None:
+            df_category = df_category[df_category["model"].isin(args.model_list)]
+        df_category["category"] = df_category["question_id"].apply(
+            lambda x: CATEGORIES[(x-81)//10]
+        )
+        df_4 = df_category[["model", "category", "score"]].groupby(["model", "category"]).mean()
```

Comment:
>df_4 [](http://example.com/codeflow?start=8&length=4)

can we also plot the radar figure, and log it as an asset in mlflow.   

in that case  users will be able to visualize the figure as well at the end out the run

Reply:
i have updated the PR to include  a) radar figure  b)  additional questions